chall01/vtran.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+import sys
+
+tittut = {
+    'a': ".-",
+    'b': "-...",
+    'c': "-.-.",
+    'd': "-..",
+    'e': ".",
+    'f': "..-.",
+    'g': "--.",
+    'h': "....",
+    'i': "..",
+    'j': ".---",
+    'k': "-.-",
+    'l': ".-..",
+    'm': "--",
+    'n': "-.",
+    'o': "---",
+    'p': ".--.",
+    'q': "--.-",
+    'r': ".-.",
+    's': "...",
+    't': "-",
+    'u': "..-",
+    'v': "...-",
+    'w': ".--",
+    'x': "-..-",
+    'y': "-.--",
+    'z': "--..",
+    ' ': " "
+}
+
+def tinnitus(str):
+    ret = ""
+    for i in str:
+        try:
+            ret += tittut[i]
+        except KeyError:
+            usage()
```

Comment:
I think for this script using try-except isn't necessary and would have been better you try to handle the input error before letting your code raise an issue and catch it to display usage.